browser/css/jssidebar.css

```diff
@@ -605,6 +607,7 @@ span.jsdialog.sidebar.ui-treeview-notexpandable {
 #TemplatePanel .sidebar .ui-content.unobutton {
 	width: var(--btn-size-m);
 	height: var(--btn-size-m);
+	background-color: transparent !important;
```

Comment:
Everything looks good, just I wonder why we not need this ^

Reply:
JsDialog `unobutton` have a set declaration on `hover`

https://github.com/CollaboraOnline/online/blob/e7901df4e740b01be9e334baa039134bfb437b26/browser/css/jsdialogs.css#L1853